pkg/lvm/lvm.go

```diff
@@ -660,6 +664,27 @@ func ignoreWarnings(str string) string {
 		if strings.HasPrefix(line, "WARNING") {
 			continue
 		}
+		// Ignore warnings of the kind:
```

Comment:
are we bumping LVM so that we can detect this particular error message?

Reply:
I thought so, but no it appears unrelated.

I'm bumping lvm2 as i'm bumping it in dcos-enterprise, really just to get the latest fixes.

Reply:
there are a couple of MESOS tickets that could be related to this:
* https://issues.apache.org/jira/browse/MESOS-4065
* https://issues.apache.org/jira/browse/MESOS-8428
* https://issues.apache.org/jira/browse/MESOS-8128